processor-kotlin/src/main/kotlin/io/datalbry/precise/processor/kotlin/visitor/FindTypesVisitor.kt

```diff
@@ -10,14 +11,19 @@ import io.datalbry.precise.api.schema.type.DocumentType
 import io.datalbry.precise.api.schema.type.EnumType
 import io.datalbry.precise.api.schema.type.Type
 import io.datalbry.precise.api.schema.type.Types
+import io.datalbry.precise.processor.kotlin.extension.*
 
 /**
  * [KSPropertyDeclaration] implementation which searches for [Type]s
  *
  * @author timo gruen - 2021-03-11
  */
-class FindTypesVisitor(private val logger: KSPLogger): KSEmptyVisitor<Unit, Set<Type>>() {
-
+class FindTypesVisitor(
+    private val logger: KSPLogger,
+    private val resolver: Resolver
+)
+    : KSEmptyVisitor<Unit, Set<Type>>()
+{
```

Comment:
This doesn't match the formatting of any other piece of code we've got.

```suggestion
): KSEmptyVisitor<Unit, Set<Type>>() {
```

Reply:
Personally I'm using that style of formatting for basically all classes with more than 1 constructor parameter) 
But we can go for the one liner if you prefer

Reply:
It's just a bit confusing, especially with the { on a new line *only* in this case